data/korath/korath weapons.txt

```diff
@@ -245,7 +245,7 @@ outfit "Korath Piercer Rack"
 outfit "Korath Piercer Launcher"
 	category "Secondary Weapons"
 	cost 593000
-	thumbnail "outfit/piercer launcher"
+	thumbnail "outfit/korath piercer launcher"
```

Comment:
It's better to remove "korath" from the piercer rack than to add "korath" to the piercer launcher.

That being said, the Korath are currently subject to a a lot of activity right now, I think it'd be better to not randomly change names of stuff that are going to subsequently be subject to more renaming in the near future. This is just asking to create conflicts and complications for them. Let's just leave all the Korath stuff to the people actively working on the Korath, and then deal with any name changes that need to be done after the dust has settled.

As far as that goes, changing the name of something that is just going to get its name changed just looks like churn.